Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.py

```diff
@@ -138,27 +136,26 @@ def get_incidents_by_keys(similar_incident_keys, time_field, incident_time, inci
     incident_time = parse_datetime(incident_time)
     max_date = incident_time
     min_date = incident_time - timedelta(hours=hours_back)
-    hours_back_query = '{0}:>="{1}" and {0}:<"{2}"'.format(time_field, min_date.isoformat(), max_date.isoformat())
+    query = ''
```

Comment:
add unit tests for the query construction if there are none

Reply:
I split the query construction part to different function with the name `build_incident_query` and added unit tests for this function.